test/run-functional-tests

```diff
@@ -0,0 +1,400 @@
+#!/bin/bash
+
+DOCKER_IMAGE="liangxin1300/hatbw"
```

Comment:
May we make it overridable `DOCKER_IMAGE=${DOCKER_IMAGE:-"liangxin1300/hatbw"}`

Reply:
Nice!